### ISSUE TITLE
[FEATURE] Add a favicon

### ISSUE THREAD

**Issue by Panquesito7** [web3community/devprotocol.xyz#19]

**Comment by krishguptadev** [comment]
I would like to work on this

**Comment by krishguptadev** [comment]
> > Yes, but it's up to Krish. Maybe u can make a lot of issues @krishguptadev (that still counts as contrib.) so they can contribute especially for hacktoberfest.
> 
> They've already made 20+ pull requests for Hacktoberfest, AFAIK.

Yeah, I leave it to hacktoberfest participants... If not assigned to someone in by tomorrow 1pm IST, I can work on it 🤔

**Comment by calvo111979** [comment]
Hi, can I work on this?

**Comment by Panquesito7** [comment]
> Hi, can I work on this?

Feel free to work on this! 🙂
